charging_station.py

```diff
@@ -0,0 +1,156 @@
+import numpy as np
+import os
+from numpy.random import randint
+
+
+## Charging Station
+
+class ChargingStation:
+
+    def __init__(self):
+        self.dt = 0.5
+        self.nb_t = int(24/self.dt)
+        self.prices = {"internal" : [],"external_purchase" : [],"external_sale" : []}
+        self.efficiency = 0.95
+        self.bill = np.zeros(self.nb_t) # Where 5e penalities will be stocked
+        self.load = np.zeros(self.nb_t) # List l4
+        self.load_battery_periode = {"fast" : np.zeros((self.nb_t,2)),"slow" : np.zeros((self.nb_t,2))} # How the player wants to charge/discharge the veicules
+        self.battery_stock = {"slow" : np.zeros((self.nb_t+1,2)), "fast" : np.zeros((self.nb_t+1,2))} # State of the batteries
+        self.nb_fast_max = 2 # Number of Stations Fasts and Lows max
+        self.nb_slow_max = 2
+        self.nb_slow = 2 # Number of Stations Fast and Slow currently used
+        self.nb_fast = 2
+        self.pmax_fast = 22
+        self.pmax_slow = 3
```

Comment:
The maximal power at the scale of the charging station is missing

Reply:
Vous voulez que je le rajoute car il peut etre utile pour le player dans sa politique car en sois je n'en ai pas besoins dans mon code je penses

Reply:
Il sera utile pour contraindre les politiques de charge possibles à l'échelle de la station. Et à noter que la valeur de cette puissance max n'est pas 2*22+2*3, sinon il n'y aurait pas de contrainte puisque les 4 VEs pourraient charger simultanément à pleine puissance, mais 40 (cf. fiche joueur).

Reply:
très bien je mets ça à jour au plus vite